sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_functions.asciidoc

```diff
@@ -709,6 +709,32 @@ optimize such partial barriers.
 _{endnote}_]
 |====
 
+==== Command Graph
+
+The functions in this section are only available if the
+link:./sycl_ext_oneapi_graph.asciidoc[
+  sycl_ext_oneapi_graph] extension is supported.
+
+|====
+a|
+[frame=all,grid=none]
+!====
+a!
+[source,c++]
+----
+namespace sycl::ext::oneapi::experimental {
+
+template <typename CommandGroupFunc>
+void submit(command_graph<graph_state::modifiable> g, CommandGroupFunc&& cgf);
+
+}
+----
+!====
```

Comment:
We've not defined any exceptions here, but I presume this would inherit the exceptions from the explicit `graph.add()` API. We should try link to that wording here.